scripts/release/shared-commands/download-build-artifacts.js

```diff
@@ -3,65 +3,56 @@
 'use strict';
 
 const {exec} = require('child-process-promise');
-const {existsSync, readdirSync} = require('fs');
-const {readJsonSync} = require('fs-extra');
+const {existsSync} = require('fs');
 const {join} = require('path');
 const {getArtifactsList, logPromise} = require('../utils');
 const theme = require('../theme');
 
-const run = async ({build, cwd}) => {
+const run = async ({build, cwd, releaseChannel}) => {
   const artifacts = await getArtifactsList(build);
-  const nodeModulesArtifact = artifacts.find(entry =>
-    entry.path.endsWith('node_modules.tgz')
+  const buildArtifacts = artifacts.find(entry =>
+    entry.path.endsWith('build2.tgz')
   );
 
-  if (!nodeModulesArtifact) {
+  if (!buildArtifacts) {
     console.log(
       theme`{error The specified build (${build}) does not contain any build artifacts.}`
     );
     process.exit(1);
   }
 
-  const nodeModulesURL = nodeModulesArtifact.url;
+  // Download and extract artifact
+  await exec(`rm -rf ./build2`, {cwd});
+  await exec(
+    `curl -L $(fwdproxy-config curl) ${buildArtifacts.url} | tar -xvz`,
+    {
+      cwd,
+    }
+  );
 
+  // Copy to staging directory
+  // TODO: Consider staging the release in a different directory from the CI
+  // build artifacts: `./build/node_modules` -> `./staged-releases`
   if (!existsSync(join(cwd, 'build'))) {
     await exec(`mkdir ./build`, {cwd});
+  } else {
+    await exec(`rm -rf ./build/node_modules`, {cwd});
   }
-
-  // Download and extract artifact
-  await exec(`rm -rf ./build/node_modules*`, {cwd});
-  await exec(`curl -L ${nodeModulesURL} --output ./build/node_modules.tgz`, {
-    cwd,
-  });
-  await exec(`mkdir ./build/node_modules`, {cwd});
-  await exec(`tar zxvf ./build/node_modules.tgz -C ./build/node_modules/`, {
-    cwd,
-  });
-
-  // Unpack packages and prepare to publish
-  const compressedPackages = readdirSync(join(cwd, 'build/node_modules/'));
-  for (let i = 0; i < compressedPackages.length; i++) {
-    await exec(
-      `tar zxvf ./build/node_modules/${compressedPackages[i]} -C ./build/node_modules/`,
```

Comment:
When this was originally added, we didn't store the entire build directory as a CI artifact. Just the npm packages.

The reason we stored each module as an individual tarball was because we ran `npm pack` to run any npm pre-processing, like filtering files out based on the `package.files` array. The build script already does this, though, so we can skip the extra step.